### ISSUE TITLE
Intellij terminal format break problem

### ISSUE THREAD

**Issue by shirohoo** [zthxxx/jovial#18]
First of all, thank you for a very nice project. 👍

I am using `M1 Mac Air`.

The terminal uses `iTerm`, and the font is applied with `Monaco`.

There is no problem with iTerm, but there is a problem with the `Intellij terminal`.

Intellij font is `Jetbrains Mono`, and the `font spacing is 1.0`.

As you can see in the image, the branch name is displayed below the time, and the last character goes to the next line.

This problem only occurs in Intellij terminals.

Since the branch name is already displayed on the top line, I think it is redundant to display it again under the time. 

What do you think about removing the branch name displayed under the time?

# iTerm
![image](https://user-images.githubusercontent.com/71188307/144347576-feebf77a-c025-49ee-9cad-cf5a088e2382.png)

# Intellij Terminal
![image](https://user-images.githubusercontent.com/71188307/144348329-a715165f-ad4f-41ef-9fd2-62bbbbf5d3b0.png)

**Comment by zthxxx** [comment]
@shirohoo  Thanks for feedback, but the second branch name (under the time) seems not caused by jovial theme.

Maybe you can show your `.zshrc` config, so that I can try to find the problem

**Comment by shirohoo** [comment]
@zthxxx 

I really thought that I might have misunderstood, so after registering the issue, I looked at it for a few more hours, but again, I was wrong. 

I am really sorry. 

It's very embarrassing, but the following code was inserted in one of the configuration files for `~.zshrc`.

```shell
ZSH git framework settings - auto completion
autoload -Uz compinit && compinit
autoload -Uz vcs_info
precmd_vcs_info() { vcs_info }
precmd_functions+=( precmd_vcs_info )
setopt prompt_subst
RPROMPT=\$vcs_info_msg_0_
PROMPT=\$vcs_info_msg_0_'%# '
zstyle ':vcs_info:git:*' formats '%b'
```

I removed that code and it works very well as expected. 

I raised the wrong issue because of my impatience. 

I am very sorry for this part.  🙏

Can I close the issue? 

Lastly, thank you so much for making such a great project. 🥰
